group2_assignment2_solution.ipynb

```diff
@@ -427,16 +452,20 @@
     "# INSERT YOUR CODE HERE\n",
     "\n",
     "\n",
-    "print()\n",
+    "print(k1_tweets[0].keys())\n",
+    "# ------Unsure if this is correct------\n",
+    "\n",
     "'''\n",
```

Comment:
The questions are pretty badly worded but I think you have understood it correctly, we could probably do a loop I guess:
```
for key in k1_tweets[0].keys():
    print(key)
```